backend/remote-state/swift/backend.go

```diff
@@ -154,6 +154,12 @@ func New() backend.Backend {
 				Description: descriptions["container"],
 			},
 
+			"state_name": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: descriptions["state_name"],
+			},
```

Comment:
Could you set a default value in here with the original value?

Reply:
Hi Paul, I actually set a default on lines 289 to 293 in keeping with the style of previous code on lines 283 to 286. Is it OK to do it that way?

Reply:
You should set the value here, the difference is that if someone then explicitly writes `tfstate.tf` as a value I believe it will create a diff, although this is a backend so doesn't have the same implications as a provider. But we should use it consistently.